doc/source/installation.rst

```diff
@@ -1,140 +1,215 @@
+.. _mamba-installation:
 
-    Try installing Build Tools for Visual Studio from
-    https://visualstudio.microsoft.com/downloads/ (scroll
-    down to the Tools for Visual Studio section).
+mamba/conda 
+===========
 
+For most users, `mamba`_ is the **recommended** way to install MDAnalysis. It is a faster drop-in 
+replacement for `conda`_ and efficiently handles dependencies. We also recommend creating a new 
+environment for MDAnalysis to ensure a clean and isolated installation.
 
-If you encounter any issues following these instructions, please
-ask for help on `GitHub Discussions (Installation)`_.
+MDAnalysis supports two common ways to install with `mamba`_:
 
-.. _`GitHub Discussions (Installation)`: https://github.com/MDAnalysis/mdanalysis/discussions/categories/installation
+**1. Recommended: Use Mambaforge**
 
-conda
-=====
+`Mambaforge`_ is a minimal `conda`_ distribution that includes `mamba`_ by default, uses the `conda-forge`_ channel 
+(preferred for MDAnalysis) and avoids clutter from `Anaconda`_ base packages.
 
-If you use ``conda`` to manage your Python environment, we highly recommend creating a new environment for MDAnalysis.
-This will ensure that you have a clean installation of MDAnalysis and its dependencies, and will not interfere with other packages you may have installed.
-We further recommend that you install and use ``mamba``, a faster drop-in replacement for ``conda``.
+To install MDAnalysis with `Mambaforge`_, follow the `mambaforge installation instructions`_ and then
+run the following commands to install MDAnalysis:
 
 .. code-block:: bash
 
-    conda create --name mdanalysis
-    conda activate mdanalysis
-    conda install -c conda-forge mamba
+    mamba create -n mdanalysis mdanalysis
+    mamba activate mdanalysis
 
-To install the latest stable version of MDAnalysis via ``conda``, use the following command. This installs all dependencies needed for full analysis functionality (excluding external programs such as `HOLE`_):
+To install the :ref:`test suite <mdanalysistests>` (optional, ~90 MB):
 
 .. code-block:: bash
 
-    mamba install -c conda-forge mdanalysis
+    mamba install -c conda-forge MDAnalysisTests
 
-To upgrade:
+To upgrade, use:
 
 .. code-block:: bash
 
     mamba update mdanalysis
 
-To install the tests:
+If you're using MDAnalysis in **JupyterLab**, install `ipywidgets`_ for progress bar support:
 
 .. code-block:: bash
 
-    mamba install -c conda-forge MDAnalysisTests
+    mamba install -c conda-forge ipywidgets
+
+**2. Alternative: Install mamba to an existing conda installation**
 
-If you intend to use MDAnalysis in JupyterLab, you will have to install
-an extra package for the progress bar in analysis classes:
+If you already use `conda`_, you can add `mamba`_ to your base environment:
 
 .. code-block:: bash
 
-    conda install -c conda-forge nodejs
-    jupyter labextension install @jupyter-widgets/jupyterlab-manager
+    conda install -c conda-forge mamba
```

Comment:
Is this relevant? I don't find it in the current documentation, but in the old documentation this was [explicitly _not recommended_](https://web.archive.org/web/20240104003900/https://mamba.readthedocs.io/en/latest/installation/mamba-installation.html#existing-conda-install-not-recommended). If we really want to provide this, I think we need to highlight that this only works (does it?) in the base environment. It is mentioned, but I don't think it is highlighted enough, and we should probably use a warning box like in the `mamba` documentation.

Reply:
It works in base and the mamba instructions had just said to _only install in base_ #405 . In our old docs we somewhere incorrectly suggested to install into the mdanalysis venv and this PR fixes that IIRC.

(Admittedly, I install mamba into any existing base that were generated years ago with anaconda and then I remove the default channel... too much hassle to find out how to do a fresh mambaforge installation.)

Reply:
Let's just remove it all. As long as there are links to installation instructions for mamba or conda/anaconda, user will find out how to do it.